### PR TITLE
Convert test to use enzyme

### PR DESCRIPTION
**Pull request by rylanc**

### CODE REVIEW

File: spec/MaskedField_spec.jsx [hunk 1]
```diff
@@ -874,21 +881,20 @@ describe('MaskedField', function() {
   });
 
   afterEach(function() {
-    ReactDOM.unmountComponentAtNode(container);
+    component.detach();
```
Comment (colinharris05) [inline spec/MaskedField_spec.jsx]: I actually haven't unmounted a component after a test before. Just curious, what purpose does this have?

**Comment by zp-alfred** [comment]
Accepted by colinharris05!

**Comment by zp-alfred** [comment]
Accepted by colinharris05!

File: spec/MaskedField_spec.jsx (hunk 1 above)
Comment (rylanc) [reply spec/MaskedField_spec.jsx]: We don’t normally have to do it, but these tests actually mount the components to the document (so they have focus and selections). That’s why I’m detaching them here.

**Pull request merged**